src/graphics/device.js

```diff
@@ -1258,6 +1258,18 @@ pc.extend(pc, function () {
             }
         },
 
+        onVertexBufferDeleted: function () {
```

Comment:
I'm not clear what this function does? It is called when one model asset is unloaded. But seems to delete everything in the device?